contracts/vaults/NonvotingVault.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.3;
+
+import "../libraries/Authorizable.sol";
+import "../interfaces/ILockingVault.sol";
+import "../interfaces/IERC20.sol";
+
+contract NonvotingVault is Authorizable {
+    // The locking gov vault
+    ILockingVault public lockingVault;
+    IERC20 public immutable token;
+
+    /// @notice Constructs this contract
+    /// @param _owner The address authorized to withdraw
+    /// @param _lockingVault The governance vault which this withdraws from
+    constructor(address _owner, ILockingVault _lockingVault) Authorizable() {
+        setOwner(_owner);
+        lockingVault = _lockingVault;
+        token = lockingVault.token();
+        lockingVault.token().approve(address(lockingVault), type(uint256).max);
+    }
+
+    /// @notice Withdraws from the locking vault
+    /// @param amount The amount to withdraw
+    /// @param destination The location to send the withdrawn tokens
+    function withdraw(uint256 amount, address destination) external onlyOwner {
+        lockingVault.withdraw(amount);
+        token.transfer(destination, amount);
```

Comment:
I would prefer the `safeTransfer`, I know here we know the token so it doesn't hurt if we use safeERC20 or ERC20, but still, I like to prefer the industry-wide standard and safe-check in place.

The same applies to the `approve()` call in the constructor.

Reply:
I don't think `safeApprove()` is industry standard and I don't it is not useful here. `safeTransfer` is more standard but is also not useful here.
Generally while both have their place I don't think we should be using them as default because they increase code complexity.

Reply:
Updated to the safeERC20 methods, and for future changes we can plan on only using the "safe" versions where necessary